src/wallet/wallet.cpp

```diff
@@ -2362,7 +2362,7 @@ std::vector<CTxDestination> CWallet::ListAddrBookAddresses(const std::optional<A
     AssertLockHeld(cs_wallet);
     std::vector<CTxDestination> result;
     AddrBookFilter filter = _filter ? *_filter : AddrBookFilter();
-    ForEachAddrBookEntry([&result, &filter](const CTxDestination& dest, const std::string& label, const std::string& purpose, bool is_change){
+    ForEachAddrBookEntry([&result, &filter](const CTxDestination& dest, const std::string& label, const std::string& purpose, bool is_change) {
```

Comment:
In 38b871da82cfc7fe74cc7f844c8c6636a0a0234c "refactor: RPC 'listlabels', encapsulate 'CWallet::ListAddrBookLabels' functionality"

nit: This change should be in 1db894772fb76d3469410683fc25f1c9721327cd "wallet: implement ForEachAddrBookEntry method"